main.cpp

```diff
@@ -458,22 +572,33 @@ Thing 10) Recording Studio
     2) record strings
     3) monthly employee salary
  */
-
-/*
-You've just defined 10 UDTs!
-4 of them are unrelated (UDTs 1-4).
-5 of them form the sub-objects that make up the 10th UDT. 
-
-MOVE them below this block comment and put them in numerical order 
-(1 - 10).
-    Do not copy. 
-    I do not want to see 2 copies of your Plain-English UDTs.  I only want to see the 1 copy below this block comment, in numerical order (1 - 10).
-
-The next thing that will happen is that I will review the 
-pseudo-code that you have written.
-Send me a DM to review your pull request when the project is ready for review.
-*/
-
+struct recordingStudio
+{
+  void recRoom();
+  void mixRoom();
+  void waitingRoom();
+  int employees;
+  void studioServerHardware();
+
+  struct v_studio
+  {
+    std::string studioName = "V STUDIO";
+    std::string owner = "Eda Seda";
+    int buildYear = 2008;
+    int totalReleasedAlbum = 341;
+    double openingTime = 13.00;
+
+    void getNewStudio();
+    void recruitStaff(int numberOfNewSraff = 3);
+    int calculateFoundationYear = 14;
+  };
+  
+  void releaseAlbum(std::string genre = "Indie Pop", double pricingOfAlbum = 14.9);
+  void recordStrings(int numberOfStringInst = 3, int artists = 2);
+  void payEmployeeSalary(int numberOfEmployer = 5, double totalSalary = 29.344);
+
+  v_studio jobsDone;
+};
```

Comment:
The 5 properties of this class are supposed to be instances of UDTs 5 - 9.
In Project 3.1d, you defined classes for each of these 5 types:
```
    1) Recording Room
    2) Mixing Room
    3) Waiting Room
    4) Employees
    5) Studio Server Hardware
```
You were supposed to write the following code:
```
struct RecordingRoom { ... };
struct MixingRoom { ... };
struct WaitingRoom { ... };
struct Employees { ... };
struct StudioServerHardware { ... };
```
Each of these classes is to have 5 member variables and 3 member functions.
then:
declare ONE instance of each class inside this 10th UDT:
```
struct RecordingStudio
{
    RecordingRoom recordingRoom;
    MixingRoom mixingRoom;
    WaitingRoom waitingRoom;
    Employees employees;
    StudioServerHardware studioServerHardware;

    //3 member functions go here...
};
```
- [ ] revise